src/main/java/seedu/address/logic/parser/TutorsPetParser.java

```diff
@@ -73,6 +74,9 @@ public Command parseCommand(String userInput) throws ParseException {
         case ListModuleClassCommand.COMMAND_WORD:
             return new ListModuleClassCommand();
 
+        case ListCommand.COMMAND_WORD:
```

Comment:
Might want to consider putting this after DeleteModuleClassCommand.COMMAND_WORD but before LinkCommand.COMMAND_WORD to maintain a fixed action sequence